.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: "\U0001F41B Bug report"
+about: Report a bug or problem with the code
+labels: bug, needs-triage
+---
+
+## Description
+<!-- Describe the bug in brief here. Please include details on WHAT is failing, what behavior you were expecting, and what behavior was exhibited -->
+
+## Reproduction Steps
+<!-- Please include ALL steps taken that lead to the bug described. Stack-traces and logs can be very helpful as well, please include those below -->
+
+### Logs
+<!-- For more info on logging, please see [here](http://docs.aws.amazon.com/sdk-for-net/v3/developer-guide/net-dg-config-other.html). -->
+
+### Environment
+<!--- Include as many relevant details about the environment where the bug was discovered -->
+* SDK Version: <!-- Info from Nuget Package -->
+* Package Version: <!-- Info from Nuget Package -->
+* OS Info: <!-- Windows 10 | OSX Mojave | Ubuntu | AmazonLinux | etc. -->
+* Build Environment <!-- Visual Studio | VSCode + Terminal `dotnet` | etc. -->
```

Comment:
Minor - missing colon, and in general question template too.

Sorry I missed in the SDK.